OpenKeychain/src/main/res/xml/preference_headers.xml

```diff
@@ -1,8 +1,11 @@
 <preference-headers xmlns:android="http://schemas.android.com/apk/res/android">
     <header
+        android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$GuiPrefsFragment"
+        android:title="@string/section_gui" />
+    <header
         android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$CloudSearchPrefsFragment"
         android:title="@string/section_cloud_search" />
     <header
         android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$AdvancedPrefsFragment"
-        android:title="@string/section_passphrase_cache" />
```

Comment:
Ah, I only now understand that the YubiKey PIN usage setting is a kind of password cache as well, but even given that, the "Use number keypad for YubiKey PIN" still seems out of place, no?

Do you think it would make sense to rename this back to "Password Cache" section and pull out the YubiKey ones into their own section? It's a very specific feature after all.